app/assets/javascripts/oxalis/model/reducers/skeletontracing_reducer_helpers.js

```diff
@@ -435,7 +436,13 @@ export function createTree(state: OxalisState, timestamp: number): Maybe<TreeTyp
       const newTreeId = _.isNumber(maxTreeId) ? maxTreeId + 1 : Constants.MIN_TREE_ID;
 
       const name = generateTreeName(state, timestamp, newTreeId);
-      const groupId = Utils.toNullable(getActiveTree(skeletonTracing).map(tree => tree.groupId));
+      const groupIdOfActiveTreeMaybe = getActiveTree(skeletonTracing).map(tree => tree.groupId);
+      const groupIdOfActiveGroupMaybe = getActiveGroup(skeletonTracing).chain(group =>
```

Comment:
Can't this also simply be `getActiveGroup(skeletonTracing).map(group => group.groupId)`? Or does this not work because groupId can be null?